decoder.go

```diff
@@ -4,14 +4,54 @@ import (
 	"fmt"
 	"image"
 	"unsafe"
+
+	"github.com/pkg/errors"
 )
 
-// #cgo pkg-config: libavcodec libavutil libswscale
-// #include <libavcodec/avcodec.h>
-// #include <libavutil/imgutils.h>
-// #include <libswscale/swscale.h>
+/*
+#cgo pkg-config: libavcodec libavutil libswscale libavformat
```

Comment:
Do we need to update the build system at all to ensure `libavformat` is installed or can we assume it will be installed if the other `libav` packages are installed?

Reply:
Added both `libavformat-dev` and `libavutil-dev` to dep list in README.